ToDo.DataAccess/ToDo.WebAPI/Controllers/DefaultController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ToDo.DataAccess.DataBase;
+using ToDo.DataAccess.Models;
+
+namespace ToDo.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DefaultController : ControllerBase
+    {
+        private readonly ToDoDbContext _context;
+
+        public DefaultController(ToDoDbContext context)
+        {
+            _context = context;
```

Comment:
WTF? You've created a repository in order to create abstraction over context. You don't need it here.

Reply:
no